docs/fastdds/dds_layer/domain/domainParticipant/domainParticipant.rst

```diff
@@ -1,5 +1,192 @@
-.. _dds_layer_domainParticiant:
+.. _dds_layer_domainParticipant:
 
 DomainParticipant
 =================
 
+A :class:`DomainParticipant` is the entry point of the application to a domain.
+Every :class:`DomainParticipant` is linked to a single domain from its creation,
+and contains all the Entities related to that domain.
+It also acts as a factory for :ref:`dds_layer_publisher_publisher`, :ref:`dds_layer_subscriber_subscriber`
+and :ref:`dds_layer_topic_topic`.
+
+The behavior of the :class:`DomainParticipant` can be modified with the QoS values
+specified on :ref:`dds_layer_domainParticipantQos`.
+The QoS values can be set at the creation of the :class:`DomainParticipant`,
+or modified later with :func:`set_qos()` member function.
+
+As an Entity, :class:`DomainParticipant` accepts a :ref:`dds_layer_domainParticipantListener`
+that will be notified of status changes on the :class:`DomainParticipant` instance.
+
+
+.. _dds_layer_domainParticipantQos:
+
+DomainParticipantQos
+--------------------
+
+:class:`DomainParticipantQos` controls the behavior of the :ref:`dds_layer_domainParticipant`.
+Internally it contains the following :class:`QosPolicy` objects:
+
++------------------------------+------------------------+----------+
+| QosPolicy class              | accessor               | Mutable  |
```

Comment:
Getter/Setter would be also fine. It's just that accessor=getter, an these are also setters
```suggestion
| QosPolicy class              | Accessor/Mutator       | Mutable  |
```